src/components/pages/article/SingleArticle.jsx

```diff
@@ -24,28 +24,44 @@ const SingleArticle = () => {
       });
   }, [article_id]);
 
+  const handleUpvote = () => {
+    patchArticleById(singleArticle.article_id, {
```

Comment:
We should feedback to the user when the patch fails. At the moment I can increase the vote while being offline 🙂